.travis.yml

```diff
@@ -1,7 +1,6 @@
+dist: trusty
 language: python
-before_install:
-  - sudo apt-get install -y libjudy-dev
-  - pip install cython
-script:
-  - python setup.py install
-  - python test/test.py
+python:
+  - 2.7
+install: pip install -r requirements.txt
```

Comment:
Travis will install these by default without this line https://docs.travis-ci.com/user/languages/python#pip but explicit is better than implicit?